### PR TITLE
fix(search): wraparound text for autosuggest entries

### PR DESCRIPTION
**Pull request by LoneRifle**
## Problem and Solution 
- Change fixed height and truncated text overflow to variable height,
  padding and text wraparound
- Take the chance to use chakra-specified theme colours where possible

## Before & After Screenshots

| Before | After |
|-|-|
| image | image |

### CODE REVIEW

**Review by LinHuiqing — APPROVED** [review]
lgtm! so the only the truncation was simply because of the height set? hahah

**Comment by LoneRifle** [comment]
> lgtm! so the only the truncation was simply because of the height set? hahah

and a bunch of other CSS attributes. see changes.

**Comment by LinHuiqing** [comment]
Ah right okay, just saw it. Could we remove the black outline while we're at the search box now?

File: client/src/components/SearchBox/SearchBox.styles.scss [hunk 1]
```diff
@@ -17,8 +16,8 @@
       flex: 1;
 
       .search-box {
-        color: $neutral-900;
-        background-color: $white;
+        color: var(--chakra-colors-neutral-900);
+        background-color: white;
         border: 1px solid var(--chakra-colors-gray-800);
```
Comment (LinHuiqing) [inline client/src/components/SearchBox/SearchBox.styles.scss]:
```suggestion
```
should be this line?

**Pull request merged**